components/Home/Bonding/requests.jsx

```diff
@@ -81,53 +80,53 @@ const getBondingProgramsRequest = async ({ isActive }) => {
   return response;
 };
 
-/**
- * returns events for the product creation
- */
-const getProductEventsFn = async (eventName, retry) => {
-  const contract = getDepositoryContract();
-  const provider = getEthersProvider();
-  const block = await provider.getBlock('latest');
```

Comment:
@kupermind you can see here that events are stored in subgraph from the latest block https://github.com/valory-xyz/autonolas-tokenomics-subgraph/pull/1/files#diff-a725ae1c058279c10ca4753a5d7090b61659b60bd36fa43d75861fabffa7d367R13